.phpcq.yaml.dist

```diff
@@ -15,6 +15,10 @@ phpcq:
     composer-require-checker:
       version: ^1.0
       signed: false
+      requirements:
+        composer-require-checker:
+          signed: false
+          version: ^4.0
```

Comment:
Warum wird hier eine spezifische Version benötigt? Eigentlich macht ja PHPCQ eine korrekte Auflösung?

Reply:
bei mir springt er nicht automatisch auf die nächste Version - das hat @discordier bei mir so eingetragen

Reply:
Ich würde es dennoch mal weglassen. Dann sollte es wohl eher ein Problem in phpcq sein, dass er hier nicht die aktuellste Version nimmt. Kenne ich aber in meinen Projekte nicht.